packages/chakra-ui-docs/components/BottomLink.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div>
+    <CFlex justify="space-between">
+      <div>
+        <CLink as="router-link" :to="prev">
+          {{ prev == '' ? "" : "Prev" }}
+        </CLink>
+      </div>
+
+      <CLink as="router-link" :to="next">
+        {{ next == '' ? "": "Next" }}
+      </CLink>
+    </CFlex>
+  </div>
+</template>
+
+<script>
+import { CLink, CFlex } from '@chakra-ui/vue'
+import { findNextAndPrevRoute } from '../utils'
+
+export default {
+  components: {
+    CLink,
+    CFlex
+  },
+  data: () => ({
+    prev: '',
+    next: ''
+  }),
+  watch: {
+    $route (to, from) {
```

Comment:
I think to improve perf, let's watch only the route path.

```suggestion
    '$route.path' (to, from) {
```